File: CodeFestival/Q51-Q60/Q54_연속되는_수.py
num = list(map(int,input().split(' ')))

num.sort()
result = 0
for i in range(0, len(num) - 1, 1):
    if num[i+1] == (num[i] + 1):
        result += 1

if result == (len(num) - 1):
    print("YES")
else:
    print("NO")

# user_input = input().split(' ')
# user_input = [int(i) for i in user_input]

# def sol(l):
#     l = sorted(l)
#     for i in range(len(l) - 1):
#         if l[i]+1 != l[i+1]:
#             return 'NO'
#     return 'YES'

# print(sol(user_input))

File: CodeFestival/Q51-Q60/Q54_연속되는_수.js
let num = prompt("숫자 스탬프를 입력하세요.").split(' ').map((n) => {
    return parseInt(n, 10);
  });
  
  num.sort(function(a, b) { // 오름차순
    return a - b;
  });
  
  result = 0
  
  for (let i=0; i<(num.length-1); i++) {
    if (num[i+1] == (num[i] + 1)) {
        result += 1;
    }
  }
  
  if (result == (num.length - 1)) {
    console.log("YES")
  }
  else {
    console.log("NO")
  }
  
  // function sol(l){
  //   l.sort((a,b) => {
  //     return a-b;
  //   });
  
  //   for (let i=0; i<l.length-1; i++){
  //     if(l[i]+1 !== l[i+1]){
  //       return 'NO';
  //     }
  //   }
  //   return 'YES';
  // }
  
  // const n = prompt('입력해주세요').split(' ').map(n => parseInt(n, 10));
  
  // console.log(sol(n));